### ISSUE TITLE
qt.conf is overwritten

### ISSUE THREAD

**Issue by Ledjlale** [linuxdeploy/linuxdeploy-plugin-qt#69]
https://github.com/linuxdeploy/linuxdeploy-plugin-qt/blob/master/src/deployment.h#L50
"Overwriting existing qt.conf file". You should not overwritting custom qt.conf. 
I have a specific configuration for my application (this is the point of qt.conf) that you remove to fit for the plugin.
I think you want to protect the hard coded paths with 
```
 ofs << "# generated by linuxdeploy-plugin-qt" << std::endl
        << "[Paths]" << std::endl
        << "Prefix = ../" << std::endl
        << "Plugins = plugins" << std::endl
        << "Imports = qml" << std::endl
        << "Qml2Imports = qml" << std::endl;
```
 but you may inform it rather than erasing all configurations.

**Comment by TheAssassin** [comment]
Interesting point. I have never heard of applications using a custom `qt.conf` before.

Would it be safe to simply append our fixes (appending to an existing value if necessary)?

**Comment by Ledjlale** [comment]
I don't think so. The [Path] is more exhaustive than Prefix, Plugins, Imports and Qml2Imports and can be before another tags (like [Platforms])
Just not erase and print some warnings was an easy workaround ;) but is not enough.
The better solution would be to do "insert or update" :  It will ensure that variables used by linuxdeploy are set in the right way.
I see something like that : First test on file. If it exists, test on [Paths] else create it like now. If not tag exists, create the section else test each fields : If one exists, replace the field else create it.

**Comment by TheAssassin** [comment]
That's what I wanted to suggest. I think with a simple INI-style parser and a few lines of code we could properly implement this.

We'd have to check if https://github.com/jtilly/inih works.

**Comment by Ledjlale** [comment]
Oups sorry, I stopped on the meaning of "append our fixes" :p

**Comment by TheAssassin** [comment]
Retrospectively I should have worded this differently, I have to admit.
